utils/cache.js

```diff
@@ -40,6 +41,15 @@ const cachePool = (opt = { maximumSize: CACHE_SIZE_MB }) => {
     }
   };
 
+  /**
+   * Remove an API Response from cacheStore.
+   * @param {string} key
+   * @returns {boolean}
+   */
+  const evict = (key) => {
+    return cacheStore.delete(key);
+  };
```

Comment:
if this is just a single line function why are we extracting it ? it can be done in place

Reply:
This is just abstraction. cachePool had no exposed api to delete a key.